zallet/src/components/json_rpc/methods.rs

```diff
@@ -255,9 +255,29 @@ pub(crate) trait Rpc {
     /// returned, even though they are not immediately spendable.
     ///
     /// # Arguments
-    /// - `minconf` (default = 1)
+    /// - `minconf`: Select outputs with at least this many confirmations (default = 1)
+    /// - `maxconf`: Select outputs with at most this many confirmations (default = unlimited).
+    /// - `include_watchonly`: Include notes/utxos for which the wallet does not provide spending
+    ///   capability (default = false).
+    /// - `addresses`: A list of addresses for which to retrieve UTXOs. For shielded addresses that
+    ///   correspond to a unified account, unspent notes belonging to that account are returned
+    ///   irrespective of whether the provided address's diversifier corresponds to the diversifier
+    ///   of the address that received the funds. If this parameter is omitted or empty, all notes
+    ///   are returned, irrespective of account. (default = None)
+    /// - `as_of_height`: Execute the query as if it were run when the blockchain was at the height
+    ///   specified by this argument. The default is to use the entire blockchain that the node is
+    ///   aware of. -1 can be used as in other RPC calls to indicate the current height (including
+    ///   the mempool), but this does not support negative values in general. A “future” height will
+    ///   fall back to the current height.
     #[method(name = "z_listunspent")]
-    async fn list_unspent(&self) -> list_unspent::Response;
+    async fn list_unspent(
+        &self,
+        minconf: Option<u32>,
+        maxconf: Option<u32>,
+        include_watchonly: Option<bool>,
+        addresses: Option<Vec<String>>,
+        as_of_height: Option<i64>,
+    ) -> list_unspent::Response;
```

Comment:
Question about `z_send_many` below:
```rust
/// Change generated from one or more transparent addresses flows to a new transparent
/// address, while change generated from a legacy Sapling address returns to itself.
/// TODO: https://github.com/zcash/wallet/issues/138
///
/// When sending from a unified address, change is returned to the internal-only
/// address for the associated unified account.
```
The address referred to here that affects the change recipient is just `fromaddress`, correct? But that is only one address, so why "from one or more transparent addresses"?

Reply:
If you specify a Unified Address, funds are taken from that UA's account, not just the UA itself (per the decisions made when introducing accounts to `zcashd`). As such, funds from multiple transparent addresses can get spent.